istioctl/cmd/istioctl/kubeinject.go

```diff
@@ -122,19 +122,24 @@ func validateFlags() error {
 }
 
 var (
-	hub                 string
-	tag                 string
-	sidecarProxyUID     uint64
-	verbosity           int
-	versionStr          string // override build version
-	enableCoreDump      bool
-	imagePullPolicy     string
-	includeIPRanges     string
-	excludeIPRanges     string
-	includeInboundPorts string
-	excludeInboundPorts string
-	debugMode           bool
-	emitTemplate        bool
+	hub                          string
+	tag                          string
+	sidecarProxyUID              uint64
+	verbosity                    int
+	versionStr                   string // override build version
+	enableCoreDump               bool
+	imagePullPolicy              string
+	statusPort                   int
+	readinessPath                string
```

Comment:
Let's not add this here as well. Annotations should be sufficient for people who want fine control.

Again - too many options, it gets confusion and very few users have the information to set this properly.

Reply:
See previous reply.  If we're going to have options, they should be consistent across all interfaces.

Reply:
This seems reasonable ... not against it.  Seems like all of this should move there, no?